cmd/native-btc/relayer_test.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"testing"
+	"time"
+
+	"github.com/gonative-cc/relayer/bitcoin"
+	"github.com/gonative-cc/relayer/dal"
+	"gotest.tools/assert"
+)
+
+func TestRelayerStart(t *testing.T) {
+	db := initTestDB(t)
+
+	config := Config{
+		DatabasePath: ":memory:",
```

Comment:
relayer config should not know about db config. Standard practice is to have few config structures and then we can compose it. If a relayer only needs one parameter, eg BitcoinNode, then just pass it as a single param.

Reply:
Done